augur/filter/include_exclude_rules.py

```diff
@@ -97,10 +97,11 @@ def _parse_filter_query(query):
     ('property', <built-in function ne>, 'value')
 
     """
-    column, value = re.split(r'!?=', query)
-    op = operator.eq
-    if "!=" in query:
-        op = operator.ne
+    split_query = re.split(r'!?=', query)
+    if len(split_query) != 2:
+        raise AugurError(f"Invalid filter query `{query}`. Hint: Queries must follow the pattern of `property=value` or `property!=value`.")
```

Comment:
+1 for @huddlej's suggestion. Example:

```suggestion
        raise AugurError(f"Invalid filter query `{query}`. Hint: Queries must follow the pattern of `property=value` or `property!=value`. `--query` supports more complex queries.")
```